spec/features/reservations_index_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
```

Comment:
probably worth keeping -- more complex permissions + functionality

cross check with controller / unit tests to really determine this